### DISCUSSION TITLE
Drastic not saving gave sames to Saves directory set by Retroarch

### DISCUSSION THREAD

**Discussion by BaseInfinity** [OnionUI/Onion#1642] [Q&A]
### Checklist

- [X] I am reporting a bug in the Onion OS - Any issues with the closed source frontend ([MainUI](https://github.com/OnionUI/MainUI-issues)), or [RetroArch](https://github.com/libretro/RetroArch) or any of the cores should be directed to their respective repositories
- [X] I have reviewed the [Onion Docs](https://onionui.github.io/docs) and did not find the answer
- [X] I have searched the existing (open and closed) issues to ensure I do not log a duplicate

### Onions OS Version

4.3.0

### Miyoo Firmware Version

Latest

### Provide a clear and concise description of the issue

I know there's a problem currently regarding saved states but I can't for the life of me figure out how to get the Saves (not states) for NDS to show up like the other games.

Is that intended? Is there a path I can change somewhere? I saw one for states but didn't see one for Saves

### Steps to Reproduce

Save a NDS game and look for it in the Retro arch Saves directory

### Provide any additional context or information that may help us investigate your issue

https://github.com/OnionUI/Onion/issues/1367

**Comment by BaseInfinity** [comment]
TLDR; Is there a way to have Drastic respect the same "Saves" path that Retroarch sets for other emulators?

**Comment by BaseInfinity** [comment]
![image](https://github.com/user-attachments/assets/b243bb97-8b39-4a95-b5d4-ea41ed30db04)
Looks like Saves are being saved to the `states\Drastic` directory, I'm under the impression that the `Save` files should not be in that directory and instead go to `saves\Drastic` or `saves\` if sorting is off

**Reply by I-Val-I** [reply]
I don't have an answer your looking for but what I am trying to find out is how to transfer save files over to my 2ds I would love to be able to use ether device to play my save file, I'm not sure how to do that, and I'm only finding .dss files not the other one

**Comment by Aemiii91** [comment]
Drastic is closed source and only allows us to set a single save/state location.

**Reply by BaseInfinity** [reply]
Ahh okay that makes sense. Okay well for my solution it's easy to change the hardcoded state path to be the same as my Retroarch Saves Path and just not use states for NDS (which im fine with, the handheld was made to save games easily)

Do you think it's worth coming up with a NDS specific solution where on "NDS App Close/Open" we copy over the saved games to the respected RetroArch Saves path (looks like extension is different so it could be targeted?). 

This way there's always at least a copy of the saved game where it should be

**Reply by BaseInfinity** [reply]
Actually  I wonder if we could abuse Symlinks and create a symlink for `.dsv` files that go to `F:\Saves\CurrentProfile\states` and `.dss` files symlink to `F:\Saves\CurrentProfile\saves`

Something along those lines; It still allows one directory to be set to keep Drastic happy but it still allows us to symlink the files into the places that `RetroArch` respects for `Saves/States`
